### DISCUSSION TITLE
No `brew` command is working.

### DISCUSSION THREAD

**Discussion by raptr0n** [Homebrew/discussions#4681] [Everyday usage]
### Output of `brew config`

```shell
No output.
```

### Output of `brew doctor`

```shell
No output.
```

### Description of issue

When I use any `brew` command, there is no output, the terminal just goes to the next line with no output of the command that I might just have entered. And neither is there any response like `brew : command not found` or anything.

<img width="565" alt="Screenshot 2023-07-31 at 2 25 44 PM" src="https://github.com/Homebrew/discussions/assets/89307791/fa4d7e7e-081a-4d1e-beab-a9074de5c3d9">

**Comment by SMillerDev** [comment]
What does `which -a brew` say?

**Reply by raptr0n** [reply]
It says:
/opt/homebrew/bin/brew
/opt/homebrew/bin/brew

**Reply by SMillerDev** [reply]
How about `cat /opt/homebrew/bin/brew | head`?

**Reply by raptr0n** [reply]
Yes, for that I got this:
```
#!/bin/bash
set -u

# Fail fast with concise message when not using bash
# Single brackets is needed here for POSIX compatibility
# shellcheck disable=SC2292
if [ -z "${BASH_VERSION:-}" ]
then
  echo "Error: Bash is required to run brew." >&2
  exit 1
```

**Reply by raptr0n** [reply]
Is there something I might be missing?

**Comment by gromgit** [comment]
What does `type brew` say?

**Reply by raptr0n** [reply]
`type brew` says this:
```
brew is /opt/homebrew/bin/brew
```

**Comment by raptr0n** [comment]
Actually, thank you guys for your responses! I just had to run the homebrew install command again, and everything started working perfectly again. Thanks so much for your responses and patience.

**Reply by osalbahr** [reply]
That’s odd. I’m guessing `brew install` implicitly did `brew update` which somehow was needed.

**Reply by raptr0n** [reply]
I didn't do the `brew install` command, I did the `curl` command that is used to install homebrew in the first place. It worked luckily!
